docs/examples/prometheus_exemplars/README.rst

```diff
@@ -0,0 +1,96 @@
+======================================================
+Prometheus Metric Exemplars with OpenTelemetry Tracing
+======================================================
+
+The full code for this example is available `on Github
+<https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/tree/main/docs/examples/prometheus_exemplars>`_.
+
+This end-to-end example shows how to instrument a Flask app with with `Prometheus
+<https://prometheus.io/>`_ metrics linked to OpenTelemetry traces using exemplars. The example
+manually adds exemplars to a Prometheus Histogram which link the metric in Cloud Monitoring to
+Spans in Cloud Trace.
+
+OpenTelemetry Python is configured to send traces to the `OpenTelemetry Collector
+<https://opentelemetry.io/docs/collector/>`_ and the Collector scrapes the python server's
+Prometheus endpoint. The Collector is configured to send everything to Google Cloud Monitoring
```

Comment:
Should we have it send metrics to GMP instead?

Reply:
Let me give it a try and see if it works

Reply:
Exemplars don't seem to be working with `googlemanagedprometheus`, I'll need to dig in or we can go ahead with this.

Reply:
Nvm got it working, PTAL